### PR TITLE
[DEV-2559] Feature - Newsroom Webhooks API

### PR DESCRIPTION
**Pull request by M4rk07**
DEV-2559

Documentation of the API [here](https://developers.prezly.com/docs/api/reference/api/API.v2.yaml/paths/~1newsrooms~1%7Bnewsroom_id%7D~1webhooks/get).

### CODE REVIEW

File: src/types/NewsroomWebhook.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default interface NewsroomWebhook {
+    id: string;
+    name: string;
+    url: string;
+    events: string[];
```
Comment (e1himself) [inline src/types/NewsroomWebhook.ts:5]: Let's introduce an enum for event names?

**Review by e1himself — APPROVED** [review]

File: src/types/NewsroomWebhook.ts (hunk 1 above)
Comment (M4rk07) [reply src/types/NewsroomWebhook.ts]: Thanks for the suggestion, done!

**Review by yuriyyakym — APPROVED** [review]
Looks good to me 👍

**Pull request merged**